operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -150,15 +155,15 @@ struct GdalDatasetWriter<P: Pixel + GdalType> {
     gdal_tiff_options: GdalGeoTiffOptions,
     gdal_tiff_metadata: GdalGeoTiffDatasetMetadata,
     output_bounds: SpatialPartition2D,
-    output_geo_transform: GeoTransform,
-    x_pixel_size: f64,
-    y_pixel_size: f64,
     _type: std::marker::PhantomData<P>,
     use_big_tiff: bool,
+    global_offset: GridIdx2D,
```

Comment:
maybe call this `window_start` and the other `window_end`? Also do you think we could already put them into a window struct like it is in the gdal source? Then it is more easy when we switch all things to pixels boxes.

Reply:
I was thinking about a struct `RasterWindow { ul: GridIdx2D, lr: GridIdx2D}`